src/test/java/org/zalando/nakadi/security/AuthenticationTest.java

```diff
@@ -220,7 +220,9 @@ public SecuritySettings securitySettings() {
             new Endpoint(GET, "/event-types/foo/partitions/bar", TOKEN_WITH_EVENT_STREAM_READ_SCOPE),
             new Endpoint(GET, "/subscriptions/foo/events", TOKEN_WITH_EVENT_STREAM_READ_SCOPE),
             new Endpoint(POST, "/subscriptions", TOKEN_WITH_EVENT_STREAM_READ_SCOPE),
-            new Endpoint(PUT, "/subscriptions/foo/cursors", TOKEN_WITH_EVENT_STREAM_READ_SCOPE));
+            new Endpoint(PUT, "/subscriptions/foo/cursors", TOKEN_WITH_EVENT_STREAM_READ_SCOPE),
+            new Endpoint(GET, "/subscriptions/foo/cursors", TOKEN_WITH_EVENT_STREAM_READ_SCOPE),
```

Comment:
Maybe the intention was to have `/cursors/something` instead of duplicated rows?